src/App.tsx

```diff
@@ -1,35 +1,15 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import Header from './layout/Header.jsx';
```

Comment:
```suggestion
import Header from './layout/Header';
```
remove the extension of the file on imports, also for the line bellow this